### ISSUE TITLE
Default insertConfig chunksize and streams are NaN

### ISSUE THREAD

**Issue by jankapunkt** [VeliovGroup/meteor-autoform-file#57]
Hey @dr-dimitru ,

I am out of knowledge here. While going through the codeof PR #56 using the debugger I found a strange behavior: When there is no `insertConfig` in the schema, the values of `chunkSize` and `streams` are always `NaN`. So I checked the part where the [insertConfig is being built](https://github.com/VeliovGroup/meteor-autoform-file/blob/master/lib/client/fileUpload.js#L55). I am totally unsure about these lines:

```javascript
  this.insertConfig    = Object.assign({}, this.data.atts.insertConfig || {});
  delete this.data.atts.insertConfig;
  this.insertConfig    = Object.assign(this.insertConfig, _.pick(this.data.atts, Object.keys(defaultInsertOpts))); 
```

Why it is picking from `data.atts`? It results always in `this.insertConfig` being `{}` but this will trigger the condition of the next two lines:

```javascript
  if (!isNaN(this.insertConfig.streams) || this.insertConfig.streams !== 'dynamic') {
    this.insertConfig.streams = parseInt(this.insertConfig.streams);
  }
  if (!isNaN(this.insertConfig.chunkSize) || this.insertConfig.chunkSize !== 'dynamic') {
    this.insertConfig.chunkSize = parseInt(this.insertConfig.chunkSize);
  }
```

because on `{}` these conditions are always true and `parseInt(undefined)` creates `NaN` to `steams` and `chunksize` are always `NaN` for empty insertConfig.

Is this intended? I doubt, because the [meteor files wiki](https://github.com/VeliovGroup/Meteor-Files/wiki/Insert-(Upload)) defines their types as `{Number|dynamic}`.

My proposal would be to either remove the `_.pick` (which also removes the dependency to underscore 👍) or to add a check in the conditionals for `streams` or `chunkSize` being not undefined.

What do you think?

**Comment by dr-dimitru** [comment]
Fix is awaiting in #59

**Comment by dr-dimitru** [comment]
Fixed by #59 
Feel free to reopen it in case if the issue is still persists on your end.
